qa/automation/CareerPortal.Tests.Scenarios/Scenarious/All_Buttons.feature

```diff
@@ -0,0 +1,44 @@
+Background:
+Given I open the https://career.quantori.com/ portal in Google Chrome
+
+Scenario 1.0: Cookie Test
```

Comment:
I did not catch the idea of the test from the title, what do you want to check here
Title should display the outcome of the test, so if you want to check that cookies were accepted and added to the user's computer, then title should be like: Cookies are added when user accepts them on the site